chain-signatures/node/src/indexer_eth.rs

```diff
@@ -408,38 +406,11 @@ fn failed_blocks_channel() -> (
     mpsc::channel(MAX_FAILED_BLOCKS)
 }
 
-type BlockNumberToHashMap = HashMap<u64, alloy::primitives::B256>;
-
-#[derive(Debug, Clone)]
-struct FinalizedEpoch {
-    pub start_block: u64,
-    pub end_block: u64,
-    pub blocks: BlockNumberToHashMap,
-}
-
 const MAX_FINALIZED_BLOCKS: usize = 1024;
-fn finalized_blocks_channel() -> (mpsc::Sender<FinalizedEpoch>, mpsc::Receiver<FinalizedEpoch>) {
+fn finalized_block_channel() -> (mpsc::Sender<u64>, mpsc::Receiver<u64>) {
```

Comment:
is there no specific type for this u64? It's kind of vague what it is but I'm assuming block height. We should have a type alias if there's nothing provided